### PR TITLE
feat: added static authentication pages

### PR DESCRIPTION
**Pull request by kedark152**
### feat: added static authentication pages

-  Added Static Login Page, Signup Page.
- Added login, signup page links in landing page.

### CODE REVIEW

**Review by RakshithBhat03 — APPROVED** [review]
- The code is clean and well structured.
- Good folder structured followed

**Review by nerdyspook — APPROVED** [review]
Clean and well structured code, looks good to merge.

File: src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              Forgot Password?
+            </Link>
+          </div>
+          {/*  Remember Me  */}
+          <div className="remember-item mg-xsm fw-bold align-center">
+            <input type="checkbox" className="mg-xsm" id="remember" />
+            <label htmlFor="terms">Remember Me</label>
+          </div>
+          {/* Buttons */}
+          <a href="" className="btn btn-solid fw-bold primary-bg-color">
```
Comment (aliamaanansari) [inline src/pages/Login.js:50]: you can use Link component from react-router instead of anchor tag

Comment (kedark152) [reply src/pages/Login.js:50]: yes, I will add it, once forgot password page is ready.

**Comment by kedark152** [comment]
Thanks all for PR Review.

**Pull request merged**